library/core/src/future/into_future.rs

```diff
@@ -130,7 +132,8 @@ pub trait IntoFuture {
     /// # }
     /// ```
     #[stable(feature = "into_future", since = "1.64.0")]
-    #[lang = "into_future"]
+    #[cfg_attr(not(bootstrap), lang = "into_future_into_future")]
```

Comment:
Renamed the existing lang item to `into_future_into_future`, and reused that name for the trait name for consistency!